lib/dockerize_rails/docker_commands/docker_options.rb

```diff
@@ -0,0 +1,58 @@
+module DockerizeRails
+  module DockerCommands
+    class DockerOptions
+      def initialize
+        @options = {
+          'Image' => '',
+          'name' => '',
+          'Hostname' => '0.0.0.0',
+          'ExposedPorts' => {},
+          'Env' => [],
+          'HostConfig' => {
+            'PortBindings' => {},
+            'Links' => []
+          }
+        }
+      end
+
+      def image(image)
+        @options['Image'] = image
+      end
+
+      def name(name)
+        @options['name'] = name
+      end
+
+      def hostname(hostname)
+        @options['Hostname'] = hostname
+      end
+
+      def expose(port)
+        @options['ExposedPorts']["#{port}/tcp"] = {}
+        @options['HostConfig']['PortBindings']["#{port}/tcp"] = [{'HostPort' => port}]
+      end
+
+      def add_port_binds(container, host)
+        @options['HostConfig']['PortBindings']["#{container}/tcp"] = [{'HostPort' => host}]
+      end
+
+      def add_env(env)
+        @options['Env'] << env
+      end
+
+      def add_links(container, alias_name)
+        @options['HostConfig']['Links'] << "#{container}:#{alias_name}"
+      end
+
+      def options
+        deep_copy @options
+      end
+
+      private
+
+      def deep_copy(o)
```

Comment:
__[reek]__
* [UncommunicativeParameterName] has the parameter name 'o' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Parameter-Name.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->